components/tag/index.css

```diff
@@ -12,195 +12,190 @@
+	--spectrum-tag-border-color-hover: var(--highcontrast-tag-border-color-hover, var(--mod-tag-border-color-hover, transparent));
+	--spectrum-tag-border-color-active: var(--highcontrast-tag-border-color-active, var(--mod-tag-border-color-active, transparent));
+	--spectrum-tag-border-color-focus: var(--highcontrast-tag-border-color-focus, var(--mod-tag-border-color-focus, transparent));
+	--spectrum-tag-border-color-emphasized: var(--highcontrast-tag-border-color-emphasized, var(--mod-tag-border-color-emphasized, transparent));
+	--spectrum-tag-border-color-disabled: var(--highcontrast-tag-border-color-disabled, var(--mod-tag-border-color-disabled, transparent));
 
 	/* background color */
-	--spectrum-tag-background-color: var(--spectrum-gray-50);
-	--spectrum-tag-background-color-hover: var(--spectrum-gray-50);
-	--spectrum-tag-background-color-active: var(--spectrum-gray-100);
-	--spectrum-tag-background-color-focus: var(--spectrum-gray-50);
-
-	/* border radius */
-	--spectrum-tag-size-small-corner-radius: var(--spectrum-corner-radius-100);
-	--spectrum-tag-size-medium-corner-radius: var(--spectrum-corner-radius-100);
-	--spectrum-tag-size-large-corner-radius: var(--spectrum-corner-radius-100);
-
-	/* border color */
-	--spectrum-tag-border-color: var(--spectrum-gray-700);
-	--spectrum-tag-border-color-hover: var(--spectrum-gray-800);
-	--spectrum-tag-border-color-active: var(--spectrum-gray-900);
-	--spectrum-tag-border-color-focus: var(--spectrum-gray-800);
+	--spectrum-tag-background-color: var(--highcontrast-tag-background-color, var(--mod-tag-background-color, var(--spectrum-gray-100)));
+	--spectrum-tag-background-color-hover: var(--highcontrast-tag-background-color-hover, var(--mod-tag-background-color-hover, var(--spectrum-gray-200)));
+	--spectrum-tag-background-color-active: var(--highcontrast-tag-background-color-active, var(--mod-tag-background-color-active, var(--spectrum-gray-200)));
+	--spectrum-tag-background-color-focus: var(--highcontrast-tag-background-color-focus, var(--mod-tag-background-color-focus, var(--spectrum-gray-200)));
+
+	--spectrum-tag-background-color-selected: var(--highcontrast-tag-background-color-selected, var(--mod-tag-background-color-selected, var(--spectrum-neutral-background-color-default)));
+	--spectrum-tag-background-color-selected-hover: var(--highcontrast-tag-background-color-selected-hover, var(--mod-tag-background-color-selected-hover, var(--spectrum-neutral-background-color-hover)));
+	--spectrum-tag-background-color-selected-active: var(--highcontrast-tag-background-color-selected-active, var(--mod-tag-background-color-selected-active, var(--spectrum-neutral-background-color-down)));
+	--spectrum-tag-background-color-selected-focus: var(--highcontrast-tag-background-color-selected-focus, var(--mod-tag-background-color-selected-focus, var(--spectrum-neutral-background-color-key-focus)));
```

Comment:
Could "selected" tokens be used for the value of existing custom properties in `.spectrum-Tag.is-selected`, instead of creating new custom properties? For example:
```
&.is-selected {
  --spectrum-tag-background-color: var(--highcontrast-tag-background-color-selected, var(--mod-tag-background-color-selected, var(--spectrum-neutral-background-color-default)));
  --spectrum-tag-content-color: var(--highcontrast-tag-content-color-selected, var(--mod-tag-content-color-selected, var(--spectrum-gray-25)));
  --spectrum-tag-border-color: var(--highcontrast-tag-border-color-selected, var(--mod-tag-border-color-selected, transparent));
}
```

Reply:
I'll have to come back to this one, I made an attempt but started getting tangled when it came to hover/focus/active states. Is button a good component to reference to see where else it's done like this?

Reply:
Done in 96247550faf1b2c87f7a9a9cb042d9d8acf539a8